account_reconcile_oca/views/account_account_reconcile.xml

```diff
@@ -135,7 +135,7 @@
         <field name="name">Reconcile</field>
         <field name="res_model">account.account.reconcile</field>
         <field name="view_mode">kanban</field>
-        <field name="domain">[("partner_id", "=", id)]</field>
+        <field name="domain">[("partner_id", "=", active_id)]</field>
```

Comment:
**Extract of wiki Migration to version 17.0:**
If you are using active_id, active_ids or active_model in your views (for example, for a smart-button context), you should replace them by the corresponding good practices:

id instead of active_id.
Hard-coded model name instead of active_model.
active_ids doesn't have direct replacement candidate. It will depend on what you want to achieve. It's a weird case though.
A warning will be issued if not replaced. Check the framework change and replacement examples at https://github.com/odoo/odoo/commit/daf05d48ac76d500aa1285184bdddc4c67641d58.

Reply:
Yes, but this is not a view, it is directly the action :wink:

Reply:
Thanks.